CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 - Create class `SpyglassGroupPart` to aid delete propagations #899
 - Fix bug report template #955
-- Add rollback option to `populate_all_common` #957
```

Comment:
I assume #9XX should be #971

Reply:
Oops, forgot to push my last commit before marking ready for review